modules/peridynamics/include/kernels/EmptyKernel.h

```diff
@@ -0,0 +1,30 @@
+//* This file is part of the MOOSE framework
```

Comment:
I was just making another pass through this code in preparation for merging, and noticed this.  MOOSE has a NullKernel, which I believe does the same thing as this.